README.md

```diff
@@ -49,7 +59,7 @@ bazel test imagick:go_default_test
 
 ## Testing that it works in production
 
-Linux
+Linux (CentOS/7)
```

Comment:
why is CentOS tested for production but Ubuntu for dev? does dev just mean the CDbuilders in this context?

Reply:
Yes, `dev` would be either developers running Linux, or build agents. The idea is to confirm cross-compilation support by building/linking in one flavor of Linux, and running on another.